borgsnap

```diff
@@ -64,17 +64,18 @@ usage() {
 }
 
 die() {
-  echo "$0: $@" >&2
+  echo "$0: $*" >&2
   exit 1
 }
 
 [[ $# == 1 ]] || usage
 
 [[ -r $1 ]] || die "Unable to open $1"
 
-source $1
+source "$1"
 
-export BORG_PASSPHRASE=$(< $PASS)
+BORG_PASSPHRASE=$(< "$PASS")
+export BORG_PASSPHRASE
```

Comment:
There's enough other bash-specific constructs elsewhere in this script that I don't see a good reason to separate export and assignment.